File: Resolvendo exercicíos em Python/secao06/Exercicio09.py
poluicao = float(input("Digite o indice: "))

if poluicao >= 0.3 and poluicao < 0.4:
    print("Grupo 1 suspender atividades!!")
elif poluicao >= 0.4 and poluicao < 0.5:
    print("Grupo 2 suspender atividades!!")
elif poluicao >= 0.5:
    print("Todos os grupos devem suspender atividdaddes")
else:
    print("Tudo certo")

File: Resolvendo exercicios em JAVA/src/secao06/Exercicio09.java
package secao06;

import java.util.Scanner;

public class Exercicio09 {

	public static void main(String[] args) {
		float indice;
		Scanner teclado = new Scanner(System.in);

		System.out.println("Informe o índice de poluição: ");
		indice = teclado.nextFloat();
		
		if(indice >= 0.3 && indice < 0.4) {
			System.out.println("Atenção: Indústrias do primeiro grupo devem suspender as atividades.");
		}else if(indice >= 0.4 && indice < 0.5) {
			System.out.println("Atenção: Indústrias do primeiro e segundo grupo devem suspender as atividades.");
		}else if(indice >= 0.5) {
			System.out.println("Atenção: Todos os grupos deevem suspender as atividades.");
		}
		
		teclado.close();
	}

}
